knimin/handlers/access_control.py

```diff
@@ -33,6 +33,8 @@ def post(self):
 
         all_levels = db.get_access_levels()
         user_levels = db.get_access_levels_user(user)
+        # TODO: I think the condition must check the access level of the logged
```

Comment:
oh. I never noticed that decorator. Looks like you are right and this if statement is not necessary at all.